integration_test.go

```diff
@@ -21,13 +21,13 @@ import (
 	"go.viam.com/rdk/pointcloud"
 	"go.viam.com/rdk/services/slam"
 	"go.viam.com/rdk/spatialmath"
-	slamConfig "go.viam.com/slam/config"
-	"go.viam.com/slam/dataprocess"
-	slamTesthelper "go.viam.com/slam/testhelper"
 	"go.viam.com/test"
 
 	viamcartographer "github.com/viamrobotics/viam-cartographer"
+	vcConfig "github.com/viamrobotics/viam-cartographer/config"
+	"github.com/viamrobotics/viam-cartographer/dataprocess"
 	"github.com/viamrobotics/viam-cartographer/internal/testhelper"
+	vcTesthelper "github.com/viamrobotics/viam-cartographer/testhelper"
```

Comment:
could we move `"github.com/viamrobotics/viam-cartographer/testhelper"` into `"github.com/viamrobotics/viam-cartographer/internal/testhelper"`